msteams-platform/m365-apps/prerequisites.md

```diff
@@ -6,91 +6,105 @@ ms.topic: how-to
+
+To set up your development environment:
 
 > [!div class="checklist"]
-> * [Obtain an M365 Developer (Sandbox) Tenant and enable sideloading](#prepare-a-developer-tenant-for-testing)
-> * [Enroll your M365 tenant in *Office 365 Targeted Releases*](#enroll-your-developer-tenant-for-office-365-targeted-releases)
+> * [Set up M365 Developer (Sandbox) Tenant and enable sideloading](#set-up-a-developer-tenant-for-testing)
+> * [Enroll M365 tenant in *Office 365 Targeted Releases*](#enroll-your-developer-tenant-for-office-365-targeted-releases)
 > * [Configure your account to access preview versions of Outlook and Office](#install-beta-office-apps-in-your-test-environment)
 > * [Switch to the Developer Preview version of Teams](#switch-to-the-developer-preview-version-of-teams)
-> * [*Optional*] [Install Teams Toolkit extension for Visual Studio Code](#install-visual-studio-code-and-teams-toolkit-preview-extension)
+> * [Install Teams Toolkit extension for Visual Studio Code](#install-visual-studio-code-and-teams-toolkit-preview-extension)[*Optional*]
 
-## Prepare a Developer Tenant for testing
+## Set up a Developer Tenant for testing
 
-If you don't already have one, create a [Microsoft 365 developer subscription](/office/developer-program/microsoft-365-developer-program-get-started) sandbox tenant or obtain a test tenant through your organization.
+To set up a developer tenant for testing:
 
-After you have a tenant, you'll need to [enable sideloading](/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading) for your tenant by signing in to [Microsoft 365 admin center](https://admin.microsoft.com) and navigating to Show All > Teams > Teams apps > Setup policies > **Global**.  Toggle on **Upload custom apps** and **Save**.
+1. Create a [Microsoft 365 developer subscription sandbox tenant](/office/developer-program/microsoft-365-developer-program-get-started) or get a test tenant through your organization.
+1. After you create a tenant, you'll need to enable sideloading for your tenant. To enable sideloading, see [enable sideloading](/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading).
 
-If you have an existing tenant, verify that sideloading is enabled by signing in to Teams and selecting **Apps**. You will see the **Upload a custom app** option if sideloading is enabled for your tenant.
+    :::image type="content" source="images/teams-sideloading-enabled.png" alt-text="Upload a custom app option" lightbox="images/teams-sideloading-enabled.png":::
 
-:::image type="content" source="images/teams-sideloading-enabled.png" alt-text="Sideloading is enabled for your tenant if you see the option to 'Upload a custom app' from the Teams 'Apps' panel":::
+> [!Note]
+> To verify if sideloading is enabled. Sign in to Teams, select **Apps** and check for **Upload a custom app** option.
```

Comment:
Change the note with the following:
To verify if sideloading is enabled,  sign in to Teams, select Apps and then check for Upload a custom app option.